projects/rocprofiler-sdk/source/lib/python/rocpd/query.py

```diff
@@ -33,6 +33,7 @@
 from . import libpyrocpd
 from .importer import RocpdImportData
 from .time_window import apply_time_window
+from .filter import apply_filter
```

Comment:
```suggestion
from .time_window import apply_time_window
```
Actually, we don't need this filter part in query.  We should not include "filter.py" at all.  Please remove it form the PR.

Reply:
removed.